Library/Homebrew/system_command.rb

```diff
@@ -63,6 +63,7 @@ def initialize(executable, args: [], sudo: false, env: {}, input: [], must_succe
     @print_stdout = print_stdout
     @print_stderr = print_stderr
     @verbose = verbose
+    @secrets = [*secrets]
```

Comment:
`Array(secrets)` will avoid creating a new array when it already is one.

Reply:
Updated